src/Travel-agent.js

```diff
@@ -1,28 +1,61 @@
 import User from './User';
+var moment = require('moment');
 
 class TravelAgent extends User {
   constructor() {
     super();
     this.id;
   }
 
-  // should return user object by passing in the user's name. this function will return the id of the user and set it to this.id
-  findUser() {
-
+  findUser(travelerName, travelerData) {
```

Comment:
Find user seems to return the id. 
Consider renaming the method to be: findUserID (Unless this is a placeholder)

Reply:
Yes, good point! I definitely meant to have it return the traveler object and forgot. I'll change that. Thanks!